server/handlers/custom-route-handler.js

```diff
@@ -44,7 +44,9 @@ function writeStaticPageResponse(res, url, page, result, { config, renderLayout,
 
 exports.customRouteHandler = function customRouteHandler(req, res, next, { config, client, loadData, loadErrorData, renderLayout, logError, seo, domainSlug }) {
   const url = urlLib.parse(req.url, true);
-  const path = req.params[0];
+  const path = req.params[0].endsWith('/') ?
```

Comment:
No regexing that shizz by `req.params[0].replace(/\/$/,'')`?

Reply:
I prefer the regex, but don't particularly care either way.

I would like to see a test for it.

Reply:
2019 people!, use `endWith`

Reply:
Will be adding a test as patch, merging as prod deploy is on priority.